scripts/pegasus_online_node_list.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+#
+# Online replica servers using minos.
+#
+
+PID=$$
+
+if [ $# -le 2 ]; then
+  echo "USAGE: $0 <cluster-name> <cluster-meta-list> <replica-task-id-list>"
+  echo
+  echo "For example:"
+  echo "  $0 onebox 127.0.0.1:34601,127.0.0.1:34602 1,2,3"
+  echo
+  exit 1
+fi
+
+cluster=$1
+meta_list=$2
+replica_task_id_list=$3
+
+pwd="$( cd "$( dirname "$0"  )" && pwd )"
+shell_dir="$( cd $pwd/.. && pwd )"
+cd $shell_dir
+
+echo "Check the argument..."
+source ./scripts/pegasus_check_arguments.sh online_node_list $cluster $meta_list $replica_task_id_list
+
+if [ $? -ne 0 ]; then
+    echo "ERROR: the argument check failed"
+    exit 1
+fi
+
+echo "Set meta level to steady..."
+echo "set_meta_level steady" | ./run.sh shell --cluster $meta_list &>/tmp/$UID.$PID.pegasus.online_node.set_meta_level
+set_ok=`grep 'control meta level ok' /tmp/$UID.$PID.pegasus.online_node.set_meta_level | wc -l`
+if [ $set_ok -ne 1 ]; then
+  echo "ERROR: set meta level to steady failed"
+  exit 1
+fi
+
+for id in $task_id_list
+do
+  echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+  minos_bootstrap $cluster replica $id
+  if [ $? -ne 0 ]; then
+    echo "ERROR: online replica task $id failed"
+    exit 1
+  fi
+  echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+  echo "sleep for 10 seconds"
+  sleep 10
+done
+
+echo "set meta.lb.only_move_primary true"
```

Comment:
echo "This remote-command tells the meta-server to ignore copying primaries during rebalancing."
echo "So the following steps only include move_primary and copy_secondary."